vscode_extension/README.md

```diff
@@ -1,48 +1,23 @@
 # Ruby Sorbet for VS Code
 
-## Features:
+This extension only works in projects that have adopted Sorbet. For
+instructions, see here:
 
-Provides Ruby editor features using the Sorbet language server:
+- <https://sorbet.org/docs/adopting>
 
-- Live typechecking
-- Types and documentation on hover
-- Go-to-definition
-- Find all references in workspace
-- Workspace symbol search
-- Autocompletion of methods, local variables, keywords, and `sig`s.
```

Comment:
I think it would be nice to keep a small blurb of features, both just as general information and as an indication of what you get with adopting Sorbet.